addon/utils/remove-script-from-dom.js

```diff
@@ -0,0 +1,9 @@
+import canUseDOM from './can-use-dom';
+
+export default function removeScriptFromDOM(script) {
+  if (!canUseDOM) return;
```

Comment:
This may be redundant due to the current places this is being used, but felt it should be checked for here too